firefox-ios/Client/Frontend/Browser/ScreenshotHelper.swift

```diff
@@ -34,11 +34,13 @@ class ScreenshotHelper {
         /// Handle home page snapshots, can not use Apple API snapshot function for this
         guard controller != nil else { return }
 
+        /// Added check for native error pages.
+        let isNativeErrorPage = controller?.contentContainer.hasNativeErrorPage ?? false
         /// If the tab is the homepage, take a screenshot of the homepage view.
         /// This is done by accessing the content view from the content container.
         /// The screenshot is then set for the tab, and a TabEvent is posted to indicate
         /// that a screenshot has been set for the homepage.
-        if tab.isFxHomeTab {
+        if tab.isFxHomeTab || isNativeErrorPage {
             if let homeview = controller?.contentContainer.contentView {
```

Comment:
We are referencing "homepage" in the comment and "homeview" in the variable name here, but really we are referring to anything that is _not_ a webpage. Could we update the comment and variable name to something more encompassing? Maybe we could also update [this comment in BVC](https://github.com/mozilla-mobile/firefox-ios/blob/7bb5da6937e9a6fa1b577c794df64d5df55df427/firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift#L169)

Reply:
Thank you for pointing this out. I will update the code.